tests/test_utils.py

```diff
@@ -361,3 +361,28 @@ def test_decode_charset_too_far_away_without_proper_alias():
             ignore_http_header_charsets=False,
             ignore_content_header_charsets=False,
         )
+
+
+@pytest.mark.parametrize(
```

Comment:
Please add those tests as well:
- `"   uNiCoDe "` => `"utf-8"`
- if you set `set_encoding_aliases({"uniCodE": "laTin1"})`, then `assert get_encoding_by_alias("unicode") == "latin1"` is still true and `assert get_encoding_by_alias("Unicode") == "latin1"` as well

Reply:
And add `assert get_encoding_by_alias("unKnoWn") == "unknown"`

Reply:
Um, the `set_encoding_aliases` never actually does any cleaning as that is done while parsing the input from the flag. If the tests involving `set_encoding_aliases` were to pass, I would have to move that logic to strip the input and lowercase it to the method rather than from the input flag. What do you think?

Reply:
Should I add this as a docstrinng instead given this method is never called except from the main? The docstring would explain that the keys and values should be lower-cased.

Reply:
You're right, then just check that `assert get_encoding_by_alias("Unicode") == "latin1"` is True in existing test case

Reply:
Already done.